hyperstart/libhyperstart/hyperstart.go

```diff
@@ -25,9 +25,10 @@ type Hyperstart interface {
 	Unpause() error
 
 	APIVersion() (uint32, error)
-	NewContainer(c *hyperstartapi.Container) error
+	CreateContainer(c *hyperstartapi.Container) error
+	StartContainer(container string) error
 	RestoreContainer(c *hyperstartapi.Container) error
-	AddProcess(container string, p *hyperstartapi.Process) error
+	ExecProcess(container string, p *hyperstartapi.Process) error
```

Comment:
the hyperstartapi here is actually hyperstart json + serial API.
We are going to covert this interface to use hyperstart grpc API directly.
But it needs more changes and needs to be suspended until the grpc API become a little more mature.
This is just for your information.